doc/source/user/basics.copies.rst

```diff
@@ -0,0 +1,135 @@
+.. _basics.copies-and-views:
+
+****************
+Copies and views
+****************
+
+The NumPy array is a contiguous block of memory consisting of two parts- the
+data buffer with the actual data elements and the metadata which contains
+information about the data buffer. The metadata includes data type, strides
+and other important information that helps manipulate the :class:`.ndarray`
+easily. See the :ref:`organization-numpy-arrays` section for a detailed look.
+
```

Comment:
I feel like we could have a short paragraph on why this document exists. Putting myself in the shoes of a new user, it is not clear to me why I should care about copies vs. views. Maybe, something like:

"When operating on NumPy arrays, it is possible to access the internal data buffer directly using a *view*, without copying data around. This ensures good performance, but can also cause unwanted problems if the user is not aware of how this works."